rest/group_test.go

```diff
@@ -0,0 +1,27 @@
+package rest
+
+import (
+	"testing"
+
+	"github.com/RocketChat/Rocket.Chat.Go.SDK/models"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRocket_GetGroupMembers(t *testing.T) {
+	rocket := getDefaultClient(t)
+	t.Run("With channel name", func(t *testing.T) {
+		general := &models.Channel{Name: "general"}
+		members, err := rocket.GetGroupMembers(general)
+		assert.Nil(t, err)
+		// are new users added to general by default?
+		assert.GreaterOrEqual(t, len(members), 1)
+
+	})
+
+	t.Run("With channel ID", func(t *testing.T) {
+		general := &models.Channel{ID: "GENERAL"}
+		members, err := rocket.GetGroupMembers(general)
+		assert.Nil(t, err)
+		assert.GreaterOrEqual(t, len(members), 1)
+	})
```

Comment:
`general` is not a group but channel. The difference is subtle, both are considered "rooms" but public ones are channels and private ones are groups. In this particular case `groups.*` endpoints do not check any public rooms, i.e. channels, which general is.

Reply:
Just wanted to give some visual example
```sh
$ culr https://open.rocket.chat/api/v1/groups.members\?roomId\=GENERAL
{"success":false,"error":"The required \"roomId\" or \"roomName\" param provided does not match any group [error-room-not-found]","errorType":"error-room-not-found"}%
$ culr https://open.rocket.chat/api/v1/channels.members\?roomId\=GENERAL -s | jq .success
true
```

Reply:
you are right. General is a channel and using it to get members of a group will eventually fail. To add test for it I can make a private group run tests on it and then delete it. As I see function to create private group exists in realtime API so I think I can use it.